rest-api-spec/src/main/resources/rest-api-spec/api/search.json

```diff
@@ -76,6 +76,10 @@
           "type" : "string",
           "description" : "Specify the node or shard the operation should be performed on (default: random)"
         },
+        "prefixed_name": {
+          "type" : "boolean",
+          "description" : "Specify whether aggregation names should be prefixed by their respective types in the response"
+        },
```

Comment:
I think we should address search templates too in this PR, it's only two more lines that make the param accepted if I am not missing anything. `SearchTemplateResponse` embeds `SearchResponse`.